symphony/lib/toolkit/class.extensionmanager.php

```diff
@@ -1027,27 +1029,15 @@ public static function about($name, $rawXML = false)
                 $required_max_version = $xpath->evaluate('string(@max)', $release);
                 $current_symphony_version = Symphony::Configuration()->get('version', 'symphony');
 
-                // Remove pre-release notes from the current Symphony version so that
-                // we don't get false erros in the backend
-                $current_symphony_version = preg_replace(array('/dev/i', '/-?beta\.?\d/i', '/-?rc\.?\d/i', '/\.0(?:\.0)?$/'), '', $current_symphony_version);
-
-                // Munge the version number so that it makes sense in the backend.
-                // Consider, 2.3.x. As the min version, this means 2.3 onwards,
-                // for the max it implies any 2.3 release. RE: #1019
-                // Also remove any .0 when doing the comparison to prevent extensions
-                // that don't use semver yet. RE: #2146
-                $required_min_version = preg_replace(array('/\.x/', '/\.0$/'), '', $required_min_version);
-                $required_max_version = preg_replace(array('/\.x/', '/\.0$/'), 'p', $required_max_version);
-
                 // Min version
-                if (!empty($required_min_version) && version_compare($current_symphony_version, $required_min_version, '<')) {
+                if (!empty($required_min_version) && \Composer\Semver\Comparator::lessThan($current_symphony_version, $required_min_version)) {
                     $about['status'][] = Extension::EXTENSION_NOT_COMPATIBLE;
                     $about['required_version'] = $required_min_version;
 
-                    // Max version
-                } elseif (!empty($required_max_version) && version_compare($current_symphony_version, $required_max_version, '>')) {
+                // Max version
+                } elseif (!empty($required_max_version) && \Composer\Semver\Comparator::lessThan($current_symphony_version, $required_max_version)) {
```

Comment:
I believe this one should be greater than?

Reply:
@brendo Fixed in 0c95af85a2. Can I get a green check mark now ? haha! Thanks buddy.